cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -73,6 +99,9 @@ spec:
             requests:
               cpu: 50m
               memory: {{ nanny_memory }}
+          volumeMounts:
+          - name: heapster-config-volume
```

Comment:
No such thing as heapster-config-volume

Reply:
Defined below, but now I've noticed duplicated "volumes:" section - maybe that's why? Will submit a fix